### ISSUE TITLE
skip-tags does not work

### ISSUE THREAD

**Issue by sepich** [2gis/k8s-handle#119]
Hello,
it seems `--skip-tags` does not work. Steps to reproduce:
```
git clone git@github.com:2gis/k8s-handle-example.git
```
Edit config.yaml, add some tag
```
staging:
  templates:
  - template: configmap.yaml.j2
  - template: deployment.yaml.j2
  - template: service.yaml.j2
    tags: manual
```
Render is empty:
```
$ docker run --rm -v `pwd`:/tmp -w /tmp 2gis/k8s-handle k8s-handle render -s staging --skip-tags manual

                         _(_)_                          wWWWw   _
             @@@@       (_)@(_)   vVVVv     _     @@@@  (___) _(_)_
            @@()@@ wWWWw  (_)\    (___)   _(_)_  @@()@@   Y  (_)@(_)
             @@@@  (___)     `|/    Y    (_)@(_)  @@@@   \|/   (_)
              /      Y       \|    \|/    /(_)    \|      |/      |
           \ |     \ |/       | / \ | /  \|/       |/    \|      \|/
            \|//    \|///    \|//  \|/// \|///    \|//    |//    \|//
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by sepich** [comment]
Related issue:
```
$ docker run --rm -v `pwd`:/tmp -w /tmp 2gis/k8s-handle k8s-handle render --skip-tags manual2 -s staging
2020-03-17 16:35:57 INFO:k8s_handle.templating:Trying to generate file from template "service.yaml.j2" in "/tmp/k8s-handle"
2020-03-17 16:35:57 INFO:k8s_handle.templating:File "/tmp/k8s-handle/service.yaml" successfully generated

                         _(_)_                          wWWWw   _
             @@@@       (_)@(_)   vVVVv     _     @@@@  (___) _(_)_
            @@()@@ wWWWw  (_)\    (___)   _(_)_  @@()@@   Y  (_)@(_)
             @@@@  (___)     `|/    Y    (_)@(_)  @@@@   \|/   (_)
              /      Y       \|    \|/    /(_)    \|      |/      |
           \ |     \ |/       | / \ | /  \|/       |/    \|      \|/
            \|//    \|///    \|//  \|/// \|///    \|//    |//    \|//
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Desired result - to have all 3 templates rendered.

**Comment by seleznev** [comment]
It should be fixed by #120. I can't reproduce it with 0.10.2/latest images anymore:

```
$ docker run --rm -v `pwd`:/tmp -w /tmp 2gis/k8s-handle k8s-handle render -s staging --skip-tags manual
2020-03-19 03:17:57 INFO:k8s_handle.templating:Trying to generate file from template "configmap.yaml.j2" in "/tmp/k8s-handle"
2020-03-19 03:17:57 INFO:k8s_handle.templating:File "/tmp/k8s-handle/configmap.yaml" successfully generated
2020-03-19 03:17:57 INFO:k8s_handle.templating:Trying to generate file from template "deployment.yaml.j2" in "/tmp/k8s-handle"
2020-03-19 03:17:57 INFO:k8s_handle.templating:File "/tmp/k8s-handle/deployment.yaml" successfully generated

                         _(_)_                          wWWWw   _
             @@@@       (_)@(_)   vVVVv     _     @@@@  (___) _(_)_
            @@()@@ wWWWw  (_)\    (___)   _(_)_  @@()@@   Y  (_)@(_)
             @@@@  (___)     `|/    Y    (_)@(_)  @@@@   \|/   (_)
              /      Y       \|    \|/    /(_)    \|      |/      |
           \ |     \ |/       | / \ | /  \|/       |/    \|      \|/
            \|//    \|///    \|//  \|/// \|///    \|//    |//    \|//
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
```
$ docker run --rm -v `pwd`:/tmp -w /tmp 2gis/k8s-handle k8s-handle render -s staging --skip-tags manual2
2020-03-19 03:18:03 INFO:k8s_handle.templating:Trying to generate file from template "configmap.yaml.j2" in "/tmp/k8s-handle"
2020-03-19 03:18:03 INFO:k8s_handle.templating:File "/tmp/k8s-handle/configmap.yaml" successfully generated
2020-03-19 03:18:03 INFO:k8s_handle.templating:Trying to generate file from template "deployment.yaml.j2" in "/tmp/k8s-handle"
2020-03-19 03:18:03 INFO:k8s_handle.templating:File "/tmp/k8s-handle/deployment.yaml" successfully generated
2020-03-19 03:18:03 INFO:k8s_handle.templating:Trying to generate file from template "service.yaml.j2" in "/tmp/k8s-handle"
2020-03-19 03:18:03 INFO:k8s_handle.templating:File "/tmp/k8s-handle/service.yaml" successfully generated

                         _(_)_                          wWWWw   _
             @@@@       (_)@(_)   vVVVv     _     @@@@  (___) _(_)_
            @@()@@ wWWWw  (_)\    (___)   _(_)_  @@()@@   Y  (_)@(_)
             @@@@  (___)     `|/    Y    (_)@(_)  @@@@   \|/   (_)
              /      Y       \|    \|/    /(_)    \|      |/      |
           \ |     \ |/       | / \ | /  \|/       |/    \|      \|/
            \|//    \|///    \|//  \|/// \|///    \|//    |//    \|//
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Thank you for reporting! Please let me known if issue still reproduces in you environment.
